TF_Service_dotNet/TouchFree Service/Connection/ClientConnection.cs

```diff
@@ -178,7 +183,7 @@ protected void ProcessHandshake(ActionCode action, string requestContent)
                 // Validation has failed because there is no valid requestID
                 response.status = "Failure";
                 response.message = "Handshaking failed. This is due to a missing or invalid requestID";
-                Debug.LogError("Handshaking failed. This is due to a missing or invalid requestID");
+                Console.Error.WriteLine("Handshaking failed. This is due to a missing or invalid requestID");
```

Comment:
I know it was like this previously, but can we not use the response.message reference as the console error? that way we only have on string to edit if we edit it?